spec/migrations/20250317000666_remove_ovirt.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+migration '20250317000666_remove_ovirt' do
+  scenarios %w[foreman] do
+    context 'tftp_dirs' do
+      let(:answers) do
+        {
+          'foreman_compute_ovirt' => {},
+        }
+      end
+
+      it 'remove compute resource ovirt' do
+        expect(migrated_answers.not_to(include("foreman_compute_ovirt")))
+      end
+
+      it 'add plugin ovirt' do
+        expect(migrated_answers.to(include("foreman_plugin_ovirt")))
```

Comment:
```suggestion
        expect(migrated_answers['foreman::plugin::ovirt']).to be true
```

this is probably the better assertion here (as you check it's true, not only present)